internal/command/artifact/download.go

```diff
@@ -53,31 +53,31 @@ func download(cfg *config.ArtifactDownload, out io.Writer, artifactPath string)
 
 	params := artifacts.
 		NewDownloadJobAttemptArtifactParams().
+		WithContext(ctx).
+		WithTimeout(2 * time.Minute).
```

Comment:
2 min seems reasonable 👍. Can tune as needed.

Reply:
Yes, it's worth noting that the default timeout in the SDK is 30 secs (I didn't know this).

Reply:
I think for large compressed files? Or videos?

Reply:
I mean, it's hard to know what's the right value.. I can increase it a little bit.

Reply:
It's done here https://github.com/signadot/cli/pull/127/commits/102b36ff3828f93c2cc2a3ba39392fe121c37d73.